src/pineko/theory.py

```diff
@@ -395,8 +395,11 @@ def fk(self, name, grid_path, tcard, pdf):
         if sv_method is None:
             if not np.isclose(xif, 1.0):
                 check_scvar_evolve(grid, max_as, max_al, check.Scale.FACT)
-        # loading ekos
+        # loading ekos to produce a tmp copy
         with eko.EKO.edit(eko_filename) as operators:
+            eko_tmp_path = operators.paths.root.parent / "eko-tmp"
```

Comment:
```suggestion
            eko_tmp_path = operators.paths.root.parent / "eko-tmp.tar"
```

Reply:
How come that it worked ?

Reply:
Most of the programs do not care so much about extensions, they are mostly for human being and editros :)

Reply:
Okay it can make sense.